src/EFCore/Metadata/Builders/CollectionNavigationBuilder.cs

```diff
@@ -151,16 +151,38 @@ private InternalForeignKeyBuilder WithOneBuilder(MemberIdentity reference)
         {
             if (SkipNavigation != null)
             {
-                throw new InvalidOperationException(
-                    CoreStrings.ConflictingRelationshipNavigation(
-                        SkipNavigation.DeclaringEntityType.DisplayName() + "." + SkipNavigation.Name,
-                        RelatedEntityType.DisplayName() + (reference.Name == null
-                                                        ? ""
-                                                        : "." + reference.Name),
-                        SkipNavigation.DeclaringEntityType.DisplayName() + "." + SkipNavigation.Name,
-                        SkipNavigation.TargetEntityType.DisplayName() + (SkipNavigation.Inverse == null
-                                                                        ? ""
-                                                                        : "." + SkipNavigation.Inverse.Name)));
+                if (((EntityType)SkipNavigation.AssociationEntityType)?
+                    .IsImplicitlyCreatedAssociationEntityType != true)
```

Comment:
```suggestion
                if (((IConventionSkipNavigation)SkipNavigation).GetConfigurationSource() == ConfigurationSource.Explicit)
```

This should fix https://github.com/dotnet/efcore/pull/21484/files/31d8567bbefb6a012cfaff7aa6adb2145422748c..e8021a3182af25122449f0133eb14ddd6d8adbea#r450544650

Reply:
Yep. That's what I tried and it doesn't work. It'll be easier to show you.